lib/decodeBySigHash.test.ts

```diff
@@ -0,0 +1,111 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { expect } from 'earljs'
+
+import { decodeBySigHash, decodeData, fetchData, parseData } from './decodeBySigHash'
+
+describe(fetchData.name, () => {
+  it('fetches correct data by signature', async () => {
```

Comment:
These tests call actual API, right? This could easily break in the future... I guess for now it's fine but I would prefer to improve it later by using something [tape](https://github.com/substack/tape) or simply injecting mock fetch